src/Simulation/Native/src/simulator/qir.hpp

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "qir_stdlib.h"
+#include <cstdint>
```

Comment:
In the other C/C++ files you include the standard headers first, then the local headers.
Here - vice versa.
Is there any specific reason for that?

Reply:
Nope, the other order makes more sense, so I will adjust here.